### ISSUE TITLE
[WebView]Android devices cannot show the popup menu as iOS does

### ISSUE THREAD

**Issue by jims57** [flutter/flutter#62547]
Is it a bug, I can't make the **Android** device shows the **Popup Menu** by either **Double Tap** or **Long Press**.

<details>
	  <summary>flutter --version</summary>
Flutter 1.21.0-6.0.pre.42 • channel master • https://github.com/flutter/flutter.git
Framework • revision 8c5c720ce6 (2 days ago) • 2020-07-28 01:07:42 -0700
Engine • revision f27729e97b
Tools • Dart 2.10.0 (build 2.10.0-0.0.dev 24c7666def)
</details>

**====Sample code as follows====**
```dart
import 'dart:async';

import 'package:flutter/material.dart';
import 'package:flutter/cupertino.dart';
import 'package:webview_flutter/webview_flutter.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: ParentWidget(),
    );
  }
}

class ParentWidget extends StatefulWidget {
  @override
  State<StatefulWidget> createState() => ParentWidgetState();
}

class ParentWidgetState extends State<ParentWidget> {
  final Completer<WebViewController> _webViewController =
  Completer<WebViewController>();

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Welcome to Flutter'),
      ),
      body: WebView(
        initialUrl: 'https://www.xiaoe-tech.com',
        javascriptMode: JavascriptMode.unrestricted,
        onWebViewCreated: (webViewController) {
          _webViewController.complete(webViewController);
        },
      ),
      floatingActionButton: FutureBuilder<WebViewController>(
        future: _webViewController.future,
        builder: (ctx, webViewController) {
          if (webViewController.hasData) {
            return FloatingActionButton(
              child: Icon(Icons.add),
              onPressed: () {
                webViewController.data.loadUrl('http://www.baidu.com');
              },
            );
          }

          return Container();
        },
      ),
    );
  }
}

```

![image](https://user-images.githubusercontent.com/1920873/88881336-bb0be100-d261-11ea-8f88-e80d11647c59.png)

**Comment by pedromassangocode** [comment]
Also found in the current `stable`.

<details>
<summary>  flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, v1.17.5, on Mac OS X 10.15.6 19G73, locale en)
    • Flutter version 1.17.5 at /Users/pedromassango/dev/SDKs/flutter_stable
    • Framework revision 8af6b2f038 (4 weeks ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

 
[!] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /Users/pedromassango/Library/Android/sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.6, Build version 11E708
    • CocoaPods version 1.9.3

[!] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[!] IntelliJ IDEA Community Edition (version 2020.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] Connected device (1 available)
    • sdk gphone x86 • emulator-5554 • android-x86 • Android 11 (API 30)
      (emulator)

! Doctor found issues in 3 categories.
```
</details>

**Comment by jims57** [comment]
@pedromassango Do you know any alternative WebView widget or package, since **WebView-Flutter** is still in preview phase.

**Comment by pedromassangocode** [comment]
I tried the [flutter_webview_plugin](https://github.com/fluttercommunity/flutter_webview_plugin) package and it worked on Android.

<img width="406" alt="Screen Shot 2020-07-31 at 08 19 03" src="https://user-images.githubusercontent.com/67424416/89010455-830aa980-d306-11ea-913c-36050d881610.png">

**Comment by jims57** [comment]
@pedromassango Thx a lot, I will try it.

**Comment by blasten** [comment]
This has been fixed in Flutter 1.22/ WebView 1.0 - use hybrid composition in the Flutter WebView plugin.

https://github.com/flutter/plugins/tree/master/packages/webview_flutter#using-hybrid-composition
